src/test/java/seedu/duke/ParserTest.java

```diff
@@ -1,31 +1,41 @@
 package seedu.duke;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import seedu.duke.command.AddCommand;
+import seedu.duke.command.Command;
+import seedu.duke.model.Item;
+import seedu.duke.model.ItemContainer;
```

Comment:
by right testing should create Stub classes for all associated class to eliminate the possibility of test failure due to error in associated classes.

Reply:
Why do you mean by this?

Reply:
Created new issue for this and will implement under v2.0 as discussed